### PR TITLE
chore: re-enable tests for Firefox related to fixed Input.dispatchMouseEvent issues

### PR DESCRIPTION
**Pull request by whimboo**
Fixes issue #7175. With the patch on https://bugzilla.mozilla.org/show_bug.cgi?id=1740798 landed in Firefox Nightly the related tests are no longer failing.

### CODE REVIEW

**Comment by whimboo** [comment]
There are more 7 tests which are passing now. \o/

**Comment by OrKoN** [comment]
Woa! it's great!

**Comment by whimboo** [comment]
@OrKoN checks for the merge of main are all fine. Now it's waiting for your review before it can auto-merge.

**Review by OrKoN — APPROVED** [review]

**Pull request merged**
